docs/markdown/changelog.raw.md

```diff
@@ -16,6 +16,34 @@ the previous line.
 - [#5094](https://github.com/PowerDNS/pdns/pull/5094): make our zone parser adhere to RFC2308 wrt implicit TTLs and add test
 -->
 
+# PowerDNS Recursor 4.0.6
+Released 6th of July 2017
+
+This release features a fix for the ed25519 verifier. This verifier hashed the message before verifying, resulting in unverifiable signatures. Also on the Elliptic Curve front, support was added for ED448 (DNSSEC algorithm 16) by using libdecaf.
+
+Besides that, this release features massive improvements to our edns-client-subnet handling, and some IXFR fixes. Note that this release changes `use-incoming-edns-subnet` to disabled by default.
+
+## Bug fixes
+- [commit c24288b87](https://github.com/PowerDNS/pdns/commit/c24288b87): rec: Use the incoming ECS for cache lookup if `use-incoming-edns-subnet` is set (Remi Gacogne)
```

Comment:
Please drop names of employees of PowerDNS. This is default on all other changelogs.